code/modules/surgery/detoxification.dm

```diff
@@ -0,0 +1,54 @@
+/datum/surgery/detox
+	name = "Detoxification"
+	steps = list(/datum/surgery_step/incise,
+				/datum/surgery_step/retract_skin,
+				/datum/surgery_step/incise,
+				/datum/surgery_step/clamp_bleeders,
+				/datum/surgery_step/detox,
+				/datum/surgery_step/close)
+
+	target_mobtypes = list(/mob/living/carbon/human, /mob/living/carbon/monkey)
+	possible_locs = list(BODY_ZONE_CHEST)
+	requires_bodypart_type = TRUE
+	replaced_by = /datum/surgery
```

Comment:
you want this null

Reply:
`Set (replaced_by) to /datum/surgery if you want to hide a "base" surgery (useful for typing parents IE healing.dm just make sure to null it out again)`